net-misc/anydesk/anydesk-6.0.1-r1.ebuild

```
@@ -58,10 +58,12 @@ QA_PREBUILT="opt/${PN}/*"
 src_install() {
 	local dst="/opt/${PN}"
 
-	dodir ${dst}
 	exeinto ${dst}
 	doexe ${PN}
 
+	# bug 706344
+	patchelf --remove-needed libpangox-1.0.so.0 "${ED}"${dst}/${PN} || die
```

```suggestion
	patchelf --remove-needed libpangox-1.0.so.0 "${ED}${dst}/${PN}" || die
```

This should all be quoted (along with everything else in the ebuild), right?

---

I just copied that from 6.0.0-r1 ebuild. Will fix it as you indicated.